extend/generic-extractor/configuration/aws-signature/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: AWS Signature
+permalink: /extend/generic-extractor/configuration/aws-signature
+---
+
+* TOC
+{:toc}
+
+Generic extractor allows signaturing requests by [**AWS**](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html).
+Signature is the process to add authentication information to your requests.
```

Comment:
využije se to například kdy?

Reply:
https://github.com/keboola/developers-docs/pull/183/commits/46dfc7b894afa06bdef80745b5d235702cf7bc1c